compiler/src/typed/env.re

```diff
@@ -968,11 +968,11 @@ let rec find_module_descr = (path, filename, env) => {
     try(IdTbl.find_same(id, env.components)) {
     | Not_found =>
       let (_, unit_source, _) = get_unit();
-      let filename = Option.value(~default=Ident.name(id), filename);
```

Comment:
There's a similar chunk of code in the `find_module` function inside this file. I'm wondering if that needs to be adjusted like this too. (I changed it locally and it didn't affect any tests at all).

Reply:
I created a branch for this in case we want it in the future.